corehq/pillows/app_submission_tracker.py

```diff
@@ -126,3 +131,89 @@ def get_sql_app_form_submission_tracker_reindexer():
         FormReindexAccessor(include_attachments=False)
     )
     return AppFormSubmissionReindexer(doc_provider, FORM_SQL, 'form_processor_xforminstancesql')
+
+
+@quickcache(['domain'], timeout=60 * 60 * 5)
+def _get_apps_for_domain(domain):
+    project = Domain.get_by_name(domain)
+    if project:
+        return [app._id for app in project.applications()]
```

Comment:
This just calls `get_brief_apps_in_domain(domain_name)` so it's not worth fetching the whole domain object.